recipes/simbac/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "simbac" %}
+{% set version = "0.1a" %}
```

Comment:
Is this an alpha release? Those aren't meant for end-users.

Reply:
The software seems to be without an active maintainer, and has never been officially versioned. What would be a suitable value for the version field? I think we can consider it to be out of alpha since it was published.

Reply:
It's fine as is then. I mostly wanted to check because sometimes people submit their own tools without giving them a proper release number first. But since this isn't your package there's nothing that can be done about it.